PublicMapping/districtbuilder
# Disable duplicate project for archived region

## Overview

- Disables the copy map button in project page for any maps with archived regions
- Removes the Duplicate Map option from the map flyout menu for any archived regions

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/130247039-42a6711a-b913-49a6-ab6a-ec24b06ffeb0.png)


### Notes

I looked into adding a tooltip to this button indicating that you first need to convert the map to a 2020 region before duplicating, but I was struggling to get the tooltip to show. It seems like the existing indicator on the map should be good, though.

## Testing Instructions

- Create a project, then set it's regionConfig to be archived
- Navigate to the project page - the copy map button should be disabled

Closes #952
